lang/LangPrimSource/SC_ComPort.cpp

```diff
@@ -205,8 +205,25 @@ UDP::UDP(int inPortNum, HandlerType handlerType, int portsToCheck): mPortNum(inP
         }
     }
 
-    boost::asio::socket_base::send_buffer_size option(65536);
-    mUdpSocket.set_option(option);
+    boost::asio::socket_base::send_buffer_size send_buffer_size;
+    mUdpSocket.get_option(send_buffer_size);
+    if (send_buffer_size.value() < 8 * 1024 * 1024) {
```

Comment:
It's not good to repeatedly hardcode numbers. Instead I would suggest to add 
```
static constexpr int sendBufferSize = 8 * 1024 * 1024;
```
to the `UDP::UDP` class and use that. If we want to change the send buffer size, we can then do it in one place.